mypy.ini

```diff
@@ -0,0 +1,4 @@
+[mypy-ruamel.*]
+ignore_errors = True
+[mypy-schema_salad.*]
+ignore_errors = True
```

Comment:
you know the reason for including ruamel here. 

There is one error which comes from schema_salad which is due to difference in the version of mypy we're using in cwltest (0.471) and schema_salad (0.520)